### PR TITLE
Media Chrome Update and hotkeys support

### PR DESCRIPTION
**Pull request by gkatsev**

### CODE REVIEW

File: packages/mux-player/src/index.ts [hunk 1]
```diff
@@ -533,6 +535,10 @@ class MuxPlayerElement extends VideoApiElement {
         }
         break;
       }
+      case PlayerAttributes.NOHOTKEYS: {
+        this.mediaController?.setAttribute(PlayerAttributes.NOHOTKEYS, newValue);
```
Comment (cjpillsbury) [inline packages/mux-player/src/index.ts:539]: question(blocker): Any reason why this isn't using the template to update (See: https://github.com/muxinc/elements/blob/main/packages/mux-player/src/media-theme-mux/media-theme-mux.ts#L77)? If not, should refactor to prefer template-based updates.

**Review by cjpillsbury — CHANGES_REQUESTED** [review]
One small callout, otherwise 👌

File: packages/mux-player/src/index.ts (hunk 1 above)
Comment (luwes) [reply packages/mux-player/src/index.ts:539]: agree here, avoid direct mutations to the DOM if possible in player

Comment (gkatsev) [reply packages/mux-player/src/index.ts:539]: Yeah, that's what I was trying to do at first, but wasn't able to get it working. Have one more thought, but may need a second set of eyes for this.

Comment (gkatsev) [reply packages/mux-player/src/index.ts:539]: Looks like it does eventually get through without this change. However, it doesn't happen until I resize the player. Thoughts on how to make sure it gets dispatched immediately?

**Review by cjpillsbury — APPROVED** [review]
👍

**Pull request merged**